src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraError.scala

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.dsde.agora.server.model
+
+import com.wordnik.swagger.annotations.{ApiModelProperty, ApiModel}
+
+import scala.annotation.meta.field
+
+/**
+ * Created by dshiga on 5/15/15.
+ */
+@ApiModel(value = "Agora Method")
+case class AgoraError(@(ApiModelProperty@field)(required = true, value = "The error message")
```

Comment:
Do we want this to be a case class with a string message, or do we want it to be an agora-specific exception, and just have that serialized to JSON? Thought I'd seen examples of spray json serializing exceptions